### PR TITLE
Return correct photo ID in ms photo importer

### PR DESCRIPTION
**Pull request by cnnorris**
@seehamrun

### CODE REVIEW

File: extensions/data-transfer/portability-data-transfer-microsoft/src/main/java/org/datatransferproject/transfer/microsoft/photos/MicrosoftPhotosImporter.java [hunk 1]
```diff
@@ -200,12 +201,21 @@ private String importSinglePhoto(
...
         if (code < 200 || code > 299) {
           throw new IOException("Got error code: " + code + " message " + response.message());
         }
-        // TODO return photo ID
-        return "fakeId";
+
+        // Extract photo ID from response body
+        if (body == null) {
+          throw new IOException("Got null body");
+        }
```
Comment (seehamrun) [inline extensions/data-transfer/portability-data-transfer-microsoft/src/main/java/org/datatransferproject/transfer/microsoft/photos/MicrosoftPhotosImporter.java]: Can we make this a precondition check instead?

File: extensions/data-transfer/portability-data-transfer-microsoft/src/main/java/org/datatransferproject/transfer/microsoft/photos/MicrosoftPhotosImporter.java [hunk 2]
```diff
@@ -200,12 +201,21 @@ private String importSinglePhoto(
             requestBuilder.header("Authorization", "Bearer " + credential.getAccessToken());
             Response newResponse = client.newCall(requestBuilder.build()).execute();
             code = newResponse.code();
+            body = newResponse.body();
```
Comment (seehamrun) [inline extensions/data-transfer/portability-data-transfer-microsoft/src/main/java/org/datatransferproject/transfer/microsoft/photos/MicrosoftPhotosImporter.java]: responseBody?

**Comment by cnnorris** [comment]
Re: album Id, we do that on lines 142-149, that's where I got  this code from

File: extensions/data-transfer/portability-data-transfer-microsoft/src/main/java/org/datatransferproject/transfer/microsoft/photos/MicrosoftPhotosImporter.java (hunk 2 above)
Comment (cnnorris) [reply extensions/data-transfer/portability-data-transfer-microsoft/src/main/java/org/datatransferproject/transfer/microsoft/photos/MicrosoftPhotosImporter.java]: good catch

**Pull request merged**